src/index.ts

```diff
@@ -107,6 +107,7 @@ export type TransferRequestBody = {
   cryptoType: CryptoType
   amount: string
   fiatAccountId: string
+  wallet?: string
```

Comment:
```suggestion
  destinationAddress?: string
```
Added a suggested rename. 

@d-mooers can you also open a PR for the [specification repo](https://github.com/fiatconnect/specification) to add this change? This repo is just for example code that helps partners implement the spec, but the spec should be the main source of truth.

Thank you for the feedback btw!!

Reply:
Will do!